src/ui/App.swift

```diff
@@ -69,91 +80,103 @@ class App: NSApplication, NSApplicationDelegate {
         // pre-load tabs so we can interact with them before the user opens the preferences window
         tabs.forEach { (tab: NSViewController) in tab.loadView() }
         preferencesWindowController = PreferencesWindowController(preferencePanes: tabs as! [PreferencePane])
+        
+        if let window = preferencesWindowController.window {
+            let quitButton = NSButton(title: NSLocalizedString("Quit", comment: ""), target: nil, action: #selector(NSApplication.terminate(_:)))
+
+            let titleBarView = window.standardWindowButton(.closeButton)!.superview!
+            titleBarView.addSubview(quitButton)
+            quitButton.translatesAutoresizingMaskIntoConstraints = false
+            let topConstraint = NSLayoutConstraint(item: quitButton, attribute: .top, relatedBy: .equal, toItem: titleBarView, attribute: .top, multiplier: 1, constant: 5)
+            let rightConstraint = NSLayoutConstraint(item: quitButton, attribute: .right, relatedBy: .equal, toItem: titleBarView, attribute: .right, multiplier: 1, constant: -5)
```

Comment:
I think you can go for `-10` here instead of `-5`. I measure on the left, the red circle is 5px from the top, and 10px from the left. I makes sense to me to mirror that